src/api/db/migrate/20200703110429_add_last_seen_at_to_notifications.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastSeenAtToNotifications < ActiveRecord::Migration[6.0]
+  def change
+    add_column :notifications, :last_seen_at, :datetime
```

Comment:
```suggestion
    add_column :notifications, :last_seen_at, :datetime, precision: 6
```

Not sure if we need to add the precision. :thinking:

Reply:
yes let's be explicit here, currently we don't rely to much on the precision, but doesn't hurt

Reply:
unless you want to do stuff  like:

```user.save
'UPDATE `users` SET `updated_at` = '2016-01-18 23:59:59.999999' WHERE `users`.`id` = 1'
```

we don't need precision at all

Reply:
@eduardoj and I have been talking about this and did some verification. Rails 6 add `precision(6)` to datetime fields created as `timestamps` and not to those created just as `datetime`.
In this case, then, I think we don't need it.